RSS Reader/RSS Reader/View Controllers/Settings/View/Resized Tableview/RSSRSelfResizedTableView.m

```diff
@@ -0,0 +1,36 @@
+//
+//  RSSRSelfResizedTableView.m
+//  RSS Reader
+//
+//  Created by Lina Loyko on 1/17/21.
+//
+
+#import "RSSRSelfResizedTableView.h"
+
+@interface RSSRSelfResizedTableView ()
+
+@property (nonatomic, assign) CGFloat maxHeight;
+
+@end
+
+@implementation RSSRSelfResizedTableView
+
+- (CGFloat)maxHeight {
+    return UIScreen.mainScreen.bounds.size.height / 2;
+}
+
+- (void)reloadData {
+    [super reloadData];
+    
+    [self invalidateIntrinsicContentSize];
+    
+    [self layoutIfNeeded];
+}
+
+- (CGSize)intrinsicContentSize {
+    CGFloat height = MIN(self.contentSize.height, self.maxHeight);
+    self.scrollEnabled = (height == self.maxHeight) ? YES : NO;
```

Comment:
just like this:
 self.scrollEnabled = (height == self.maxHeight);

Reply:
Done